python/nano/example/pytorch/quantization/inc/resnet18_cifar.py

```diff
@@ -158,4 +163,44 @@ def configure_optimizers(self):
 )
 
 trainer.fit(model, cifar10_dm)
-trainer.test(model, datamodule=cifar10_dm)
+
+fp32_model = model
+# FP32 testing
+start = time()
+outputs = trainer.test(fp32_model, datamodule=cifar10_dm)
+time1 = time() - start
+log_dict = outputs[0]
+print("FP32 model size: %.3f MB" % fp32_model.model_size)
+print("Throughput: %d s" % time1)
+print("Accuracy: top1=%.2f%%" % (log_dict['test_acc'] * 100))
+
+# Save FP32 checkpoint
+trainer.save_checkpoint('ResNet18_on_CIFAR_FP32_CKPT.pth')
```

Comment:
why do you need to save here?

Reply:
It was to check the model size. No need any more. I will remove this.